apps/web/src/pages/settings/tabs/InAppCenterCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Prism } from '@mantine/prism';
 import { colors, Text } from '../../../design-system';
-import { WIDGET_EMEBED_PATH } from '../../../config';
+import { WIDGET_EMBED_PATH } from '../../../config';
```

Comment:
Rollback this one please. This can affect current users settings and break their systems.